sbol2/property.py

```diff
@@ -823,10 +820,6 @@ def removeOwnedObject_str(self, uri):
                         # Erase TopLevel objects from Document
                         if self._sbol_owner.getTypeURI() == SBOL_DOCUMENT:
                             del obj.doc.SBOLObjects[rdflib.URIRef(uri)]
-                        # Erase nested, hidden TopLevel objects from Document
-                        if obj.doc is not None and obj.doc.find(uri) is not None:
-                            obj.doc = None  # TODO not sure what this does
-                        self.validate(None)
```

Comment:
I think you deleted one too many lines. I think we want this call to validate. Am I mistaken?

Reply:
Yep that was an unintentional merge foul